src/ralph/virtual/tests/test_models.py

```diff
@@ -0,0 +1,88 @@
+from ralph.assets.models.assets import ServiceEnvironment
+from ralph.assets.models.choices import ComponentType
+from ralph.assets.models.components import ComponentModel, GenericComponent
+from ralph.assets.tests.factories import EnvironmentFactory, ServiceFactory
+from ralph.tests import RalphTestCase
+from ralph.virtual.models import (
+    CloudFlavor,
+    CloudHost,
+    CloudProject,
+    CloudProvider
+)
+
+
+class OpenstackModelsTestCase(RalphTestCase):
+    def setUp(self):
+        self.envs = EnvironmentFactory.create_batch(1)
```

Comment:
just `self.envs = EnvironmentFactory()` - you don't have to call `create_batch` if you're creating only single instance